knock/src/core/ui/Icon/Icon.tsx

```diff
@@ -1,15 +1,15 @@
 import { Fragment } from 'react/jsx-runtime';
-import styles from '../../styles/icon/icon.module.scss';
 
 type IconProps = {
   src: string;
   alt: string;
+  className: string;
 };
 
-const Icon: React.FC<IconProps> = ({ src, alt }) => {
+const Icon: React.FC<IconProps> = ({ src, alt, className }) => {
   return (
     <Fragment>
```

Comment:
import 하지 않고 단축해서 사용할 수 있습니다! 다음부터 참고해보세요 👍 
```suggestion
    <>
```